static/js/constants.js

```diff
@@ -24,6 +24,16 @@ export const EDUCATION_LEVELS = [
   {value: DOCTORATE, label: "Doctorate"}
 ];
 
+export const FILTER_NAMES = {
```

Comment:
That's true. The reason for 'title' is that searchkit uses the title prop of the component for the title of the facet as well as the label for the selected filter. Also, I think it makes it clear that the title of the facet is used to label the selected filter.